### ISSUE TITLE
Is the Cocoa text system accounted for?

### ISSUE THREAD

**Issue by leoheitmannruiz** [SafeExamBrowser/seb-mac#380]
Hey, I hope you're doing good :)

I just wanted to make sure that you are aware of the [Cocoa text system](https://en.wikipedia.org/wiki/Cocoa_text_system) (also see [here](https://github.com/jrus/cocoa-text-system/blob/master/cocoa-text.md)).

I don't know how to test this, as I don't have a configuration file lying around, though opening https://sourceforge.net/p/seb/discussion/ in SEB, I could use my custom Cocoa text system shenanigans in the input field. The most obvious use case (to me) is the insertion of arbitrary predefined text.

Please excuse me, in case this is already cared for in proper exams.

**To Reproduce**
1. Create a ~/Library/KeyBindings/DefaultKeyBinding.dict (feel free to use mine below)
2. Type ;secret

```
{
  ";" = {
    " " = ("insertText:", ";");
    "s" = {
      "e" = { 
        "c" = {
          "r" = {
            "e" = {
              "t" = ("insertText:", "mega sick cheat, whoaaaaaaaaa!"); 
            };
          };
        };
      }; 
    };
  };
}
```

**Expected behavior**
I expect this not to work in a proper exam.

**Comment by danschlet** [comment]
Thank you for reporting this, I am looking into it.

**Comment by danschlet** [comment]
I was able to reproduce the issue, thank you for the bug report.

I will add a fix to SEB 3.3.3 for macOS.

**Comment by danschlet** [comment]
This should be fixed in [SEB 3.3.3 for macOS](https://github.com/SafeExamBrowser/seb-mac/releases/tag/3.3.3). Please test and report if it works for you.
